qcodes_contrib_drivers/drivers/Tektronix/Keithley_6500.py

```diff
@@ -2,55 +2,154 @@
 from qcodes.instrument import InstrumentChannel
```

Comment:
It's better to import from the full path: from qcodes.instrument.channel import InstrumentChannel